src/pages/ZapConfig.vue

```diff
@@ -311,17 +311,15 @@ export default {
           this.selectedZclGenData.length != 0
         ) {
           let data = {
```

Comment:
This file has this code: 
`  beforeCreate() {
    // setCssVar('primary', '#33F')
    this.$serverGet(restApi.uri.initialPackagesSessions).then((result) => {
      this.zclPropertiesRow = result.data.zclProperties
      this.selectedZclPropertiesData = result.data.zclProperties[0]
      this.zclGenRow = result.data.zclGenTemplates`

Which is the first REST API call. If session creation is in http-server then the session would be created by this REST API, I believe.